bokehjs/src/coffee/models/widgets/main.coffee

```diff
@@ -1,34 +1,36 @@
 module.exports = {
-  editors:                  [require('./cell_editors'), 'Editor']
-  formatters:               [require('./cell_formatters'), 'Formatter']
+  models: {
```

Comment:
I agree, I am just trying to develop habits of extreme fastidiousness around any changes.